Maple2Storage/Types/Metadata/ItemMetadata.cs

```diff
@@ -223,6 +213,85 @@ public ItemBreakReward(int id, int count)
         public override string ToString() => $"Id: {Id}, Amount: {Count}";
     }
 
+    [XmlType]
+    public class Function
```

Comment:
`Function` probably would be confuse with something else, perhaps `ItemFunction` or `ItemMetadataFunction`